internal/exec/exec.go

```diff
@@ -0,0 +1,28 @@
+package exec
+
+import (
+	"bytes"
+	"git.sr.ht/~spc/go-log"
+	"os/exec"
+)
+
+func RunProcess(command string, args []string, env []string) error {
+	cmd := exec.Command(command, args...)
```

Comment:
I wonder whether something similar to https://github.com/juntaki/bucketsync would make more sense than running a command?

Reply:
I've used https://github.com/seqsense/s3sync instead.

Reply:
I suggested this project because the idea is interesting and it would run in process. Managing external processes is a bit more tricky to do it well.

Reply:
As far as I understand the project you suggested is for mounting s3 to local fs and we would have to reconcile files on our own. Both original in-OS `aws s3 sync` and the library I eventually used instead to that for us.

Reply:
OK.. After some thinking I think I understand how it could be used as it is...